### ISSUE TITLE
python

### ISSUE THREAD

**Issue by simeond** [Acidham/alfred-markdown-notes#16]
Hi there

I've started having problems with using the md* commands.  Typing "md" brings up the list of options, but neither "mds" nor "mdt" search my notes anymore and mdc doesn't create a note.

this is info from debugging traceback, which seems to suggest that it import name 'readPlist' from 'plistlib'.   It looks like it is searching for python 3.9 at 'cellar' location.  But this is neither my active python distribution (anaconda) nor listed if I type 'where python'. 

```
[07:33:59.434] Markdown Notes[Script Filter] Queuing argument ''
[07:33:59.621] Markdown Notes[Script Filter] Script with argv '' finished
[07:33:59.623] ERROR: Markdown Notes[Script Filter] Code 1: Traceback (most recent call last):
  File "/Users/simeonduckworth/Dropbox/id_kit/Alfred.alfredpreferences/workflows/user.workflow.B78B6C3A-AABD-4027-B027-41494DEB2E2A/notes_search.py", line 5, in <module>
    from Alfred3 import Items as Items
  File "/Users/simeonduckworth/Dropbox/id_kit/Alfred.alfredpreferences/workflows/user.workflow.B78B6C3A-AABD-4027-B027-41494DEB2E2A/Alfred3.py", line 7, in <module>
    from plistlib import readPlist, writePlist
ImportError: cannot import name 'readPlist' from 'plistlib' (/usr/local/Cellar/python@3.9/3.9.0_1/Frameworks/Python.framework/Versions/3.9/lib/python3.9/plistlib.py)
```

I can't pinpoint whether this is was when I updated to Big Sur or (more likely) following a python update.  But I cant figure out how to change the configuration (or what to change it to!)

Not sure if others have the same problem - or it is just me!  Very grateful for any help

**Comment by Acidham** [comment]
Please try version 3.1.1 from here: https://github.com/Acidham/alfred-markdown-notes

**Comment by simeond** [comment]
hi there

Thanks for the prompt reply.

I updated to 3.1.1 but i get exactly the same issue.  Restarted alfred, still the same.




---
Simeon Duckworth
+ 447979917242

**Comment by Acidham** [comment]
I forgot to mention, after you upgraded the workflow please delete following file: `~/Library/Application Support/Alfred/Workflow Data/com.apple.alfred.workflow.mdnotes/py3`

This enforces that the workflow reinitialize the python version.

**Comment by simeond** [comment]
that worked! thanks




---
Simeon Duckworth
+ 447979917242

**Comment by Acidham** [comment]
Finally, fixed in v3.1.2

**Comment by simeond** [comment]
thanks -- i've downloaded the latest version and it all works fine.
(love using this workflow, btw.  great work)




---
Simeon Duckworth
+ 447979917242
